docs/general/events.md

```diff
@@ -17,74 +17,29 @@ network connection severed, etc.
-  and the set of attributes.
-* The _payload_ (body) MAY contain any type supported by the OpenTelemetry data
-  model for the log [body](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-body)
-  and the semantic conventions will define the expected structure of the _payload_
-  (body) for the event.
-* The _payload_ (body) SHOULD be used to represent the structure of the event.
-
-Recommendations for defining events:
-
-* Use the _payload_ (body) to represent the details of the event instead of a
-  collection of [standard](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.45.0/specification/common#attribute)
-  attributes.
-* Events SHOULD be generated / produced / recorded using the
-    [Logs API](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.45.0/specification/logs/api.md#emit-a-logrecord)
-    to ensure that the event is created using the configured SDK instance.
-  * The Emit Event API is not yet available in all OpenTelemetry SDKs. Check [spec-compliance matrix](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.45.0/spec-compliance-matrix.md#logs) to see the implementation status in corresponding language.
-* It's NOT RECOMMENDED to prefix the _payload_ (body) _fields_ with the `EventName` to
-  avoid redundancy and to keep the event definition clean.
-* The events SHOULD document their semantic conventions including event name,
-  attributes, and the payload.
-
-Recommendations on using attributes vs. body fields:
-
-* If the field should be comparable across events with different `EventName` (or between an event and other telemetry items),
-  it should be an attribute.
-* If the field is specific to the event itself, then it should be a body field.
-* Body fields that belong to events with different event names are not comparable.
-  For example, body field `id` on event `my_company.order_submitted` is semantically different from
-  field `id` on an event with name `session.start`.
+  that uniquely identifies the event. Event names are subject to the [Naming guidelines](/docs/general/naming.md). Event name SHOULD be of a low-cardinality.
+* Events MAY have [attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.45.0/specification/logs/data-model.md#field-attributes)
+  that provide additional context about the event.
+* Events MAY contain a [body](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.45.0/specification/logs/data-model.md#field-body) of type [`any`](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.45.0/specification/logs/data-model.md#type-any).
+* The event name uniquely identifies event structure: the set of attributes and
+  the type of the body.
+* Events MAY have [severity number](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.45.0/specification/logs/data-model.md#field-severitynumber).
```

Comment:
```suggestion
* Events SHOULD have [severity number](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.45.0/specification/logs/data-model.md#field-severitynumber).
```
and move below 

> * An event MUST have an ...

Related to: https://github.com/open-telemetry/opentelemetry-specification/pull/4509

Reply:
I'd prefer to keep MAY here at least for the time being - none of the existing events have severity as of today and the tooling does not support setting it - https://github.com/open-telemetry/semantic-conventions/issues/828. 
I also have some doubts that all events should have fixed severity (e.g. resource provisioning event may be Info when successful, warning if retriable, error if completely failed)